### PR TITLE
Add justification for constraints

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Related: https://github.com/thoth-station/core/issues/271
Related: https://github.com/thoth-station/adviser/pull/1826

## This introduces a breaking change

- [x] No

### CODE REVIEW

**Comment by fridex** [comment]
/assign @KPostOffice 
PTAL.

File: _j/constraints.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+title: Removing a package based on a constraint
+---
+
+One or multiple packages were not considered during the resolution based on
+constraints supplied.
+
+## Issue description
+
+Constraints provide a mechanism to state dependencies that are known to be
+working without a need to adjust direct dependency requirements. This allows to
```
Comment (KPostOffice) [inline _j/constraints.md:11]: "This permits restricting requirements..."

**Review by KPostOffice — COMMENTED** [review]
Thanks :). Good explanation

File: _j/constraints.md (hunk 1 above)
Comment (fridex) [reply _j/constraints.md]: Thanks for the review! 👍🏻 Adjusted in c0b9dcde

**Pull request merged**